project-code/core/common/src/main/scala/play/core/server/servlet/RequestHandler.scala

```diff
@@ -102,6 +102,10 @@ trait HttpServletRequestHandler extends RequestHandler {
       case (name, value) if name.equalsIgnoreCase(play.api.http.HeaderNames.SET_COOKIE) =>
         getServletCookies(value).foreach(httpResponse.addCookie)
 
+      case (name, value) if name.equalsIgnoreCase(HeaderNames.TRANSFER_ENCODING) && value == HttpProtocol.CHUNKED =>
+        // ignore this header
+        // the JEE container sets this header itself. Avoid duplication of header (issues/289)
```

Comment:
Are we sure this header is set by all containers ?

Why did that work before ?
More, before what ? With Play 2.1.x ? Or with earlier versions of Tomcat ?